docs/core/tools/dotnet-format.md

```diff
@@ -0,0 +1,146 @@
+---
+title: dotnet format command
+description: The dotnet format command formats code to match `editorconfig` settings for the current directory.
```

Comment:
Markdown formatting (backticks in this case) doesn't work in the YAML metadata block.  Also I'd follow the precedent of the VS docs which tend to prefer EditorConfig with no formatting.  Occasionally they refer to the *.editorconfig* file in Markdown, i.e., as a file name with [italic style rather than code style](https://review.docs.microsoft.com/en-us/help/contribute/text-formatting-guidelines?branch=master#italics). 

```suggestion
description: The dotnet format command formats code to match EditorConfig settings for the current directory.
```